### ISSUE TITLE
Documentation to improve setup

### ISSUE THREAD

**Issue by hyamanieu** [OptimistikSAS/OIBus#1816]
This is neither a bug report or a feature request, just about misaligned documentation. 

I believe your documentation should be clearer about the exact environment it works, and also which path should be followed to set it up. There are 3 sources: the main [readme](https://github.com/OptimistikSAS/OIBus/blob/develop/README.md), the [developer-readme.md](https://github.com/OptimistikSAS/OIBus/blob/develop/local-development/developer-readme.md), and your website https://optimistik.com/start-with-oibus/ .

I have tried to set-up this application with three methods:

1. setting it up from source on Ubuntu 20.04 WSL
2. setting it up from your download bash script on Ubuntu 20.04 WSL
3. setting it up from your download exec on Windows 10

# 1 source on WSL U20.04

I have set up node v16.15.1 & npm v8.13.1

I couldn't build with NPM, there seems to be an incompatibility between react 18.1.0 & react-json-view 1.21.3

It was also not clear whether we should install from the master branch or the develop branch. By default, develop is cloned which seems unusual. Reading a couple PR, it was not really clear how your git flow works.

# 2 download on WSL U20.04

I downloaded following:

```bash
wget https://github.com/OptimistikSAS/OIBus/releases/latest/download/OIBus-linux.zip
unzip ./OIBus-linux.zip OIBus-linux
```

I understood quickly this won't work, as systemd doesn't exist on WSL (I have learned something). After some tweaking with https://github.com/DamionGans/ubuntu-wsl2-systemd-script.git , I could run the bash script but the service won't start:
```
>> sudo systemctl status oibus.service
Jun 28 16:13:15 wslmachine systemd[1]: oibus.service: Scheduled restart job, restart counter is at 10.
Jun 28 16:13:15 wslmachine  systemd[1]: Stopped OIBus Client.
Jun 28 16:13:15 wslmachine  systemd[1]: oibus.service: Start request repeated too quickly.
Jun 28 16:13:15 wslmachine  systemd[1]: oibus.service: Failed with result 'exit-code'.
Jun 28 16:13:15 wslmachine  systemd[1]: Failed to start OIBus Client.
```

Well it seems it's not compatible with WSL (which is not an issue, it's not the target environment)

# 3 Download and install on Windows 10

the third one did not match documentation as explained in https://optimistik.com/start-with-oibus/ . Actually your executive file does everything listed there. The service started and I could access the application.

# What I want to try at some point:

- setting it up from source on a "real" Ubuntu host or eventually on a node docker container (which would be in my case the real environment).

**Comment by hyamanieu** [comment]
Note it isn't a real Ubuntu host, but a windows subsystem Linux

**Comment by burgerni10** [comment]
Hi @hyamanieu,

We've changed quite a lot the OIBus architecture and updated our dependencies. The building should work now and we've tested OIBus on Windows, Linux and MacOS. However we don't test OIBus on WSL but we will be happy to have a PR to make it works on WSL if it's still not the case.

We also improved our documentation now available on this website: https://oibus.optimistik.com/

I close this issue for now
